service-tests/php-eye.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'php-eye', title: 'PHP version from PHP-Eye' });
+module.exports = t;
+
+t.create('gets the package version of symfony')
+    .get('/symfony/symfony.json')
+    .expectJSON({ name: 'Tested', value: '7.1' });
```

Comment:
It's good that we have a test for the badge without a version because it ensures that code path is exercised. However I'm concerned this test will be brittle. Could you use a regex here instead of a constant `'7.1'`?